test/IRGen/objc_extensions.swift

```diff
@@ -34,24 +34,20 @@ import objc_extension_base
 }
 
 extension NSObject {
-  func someMethod() -> String { return "Hello" }
+  @objc func someMethod() -> String { return "Hello" }
 }
 
 extension Gizmo: NewProtocol {
-  func brandNewInstanceMethod() {
+  @objc func brandNewInstanceMethod() {
   }
 
-  class func brandNewClassMethod() {
+  @objc class func brandNewClassMethod() {
   }
 
   // Overrides an instance method of NSObject
   override func someMethod() -> String {
     return super.someMethod()
   }
-
-  // Overrides a class method of NSObject
```

Comment:
Can you override a different class method instead? `isSubclass(of:)`, maybe?

Reply:
`isSubclass(of:)` does not appear to be overridable, but I defined another method in an extension and added an override, which has the same effect since `@objc` methods in extensions are dynamic.

Reply:
:-/ That's not exactly the same test either, since the thing being overridden isn't imported. But I guess we have a lot of other tests for overrides.